### PR TITLE
Adding DNAnexus_data_structure.md

### PR DESCRIPTION
**Pull request by arangrhie**
Here goes the .md version of the data structure.

### CODE REVIEW

File: DNAnexus_data_structure.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Assembly data structure on DNAnexus
+
+
+## Guidance
+* Each bulletin points are folder names,
+  * with each daughter folders/files shown bellow
```
Comment (mcshane) [inline DNAnexus_data_structure.md]: typo: bellow -> below

File: DNAnexus_data_structure.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+Once we visit a VGP project, the top level folders are expected to have:
+* genomic_data
+* assembly_\[ver\]
+* transcriptomic_data
+
+## Full sub-folders and files
+
+* genomic_data
+  * pacbio
+    * bam
```
Comment (mcshane) [inline DNAnexus_data_structure.md]: We talked briefly with Brett about separating the subreads and scraps BAMs into separate folders to help a bit with selecting inputs to DNAnexus pipelines...

File: DNAnexus_data_structure.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        ```
+        subreads.bam
+        subreads.bam.pbi
+        subreads.bam.bai
```
Comment (mcshane) [inline DNAnexus_data_structure.md]: Don't think there will be a `.bai` file for unsorted/unmapped data

File: DNAnexus_data_structure.md (hunk 1 above)
Comment (arangrhie) [reply DNAnexus_data_structure.md]: Thanks! Changed it.

File: DNAnexus_data_structure.md (hunk 2 above)
Comment (arangrhie) [reply DNAnexus_data_structure.md]: Have changed the structure. Let me know what you think.

**Pull request merged**